.github/workflows/release.yml

```
@@ -64,6 +93,12 @@ jobs:
         COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
         LIB_VERSION: ${{ github.event.release.tag_name }}
 
+  Deploy-Docs:
+    name: Release Backpack Docs
+    runs-on: macos-11
+    environment: Publishing
+    needs: [Deploy-SwiftUI, Deploy-UIKit, Deploy-Commons]
```

If the above suggestions go in then this would be

```suggestion
    needs: [Deploy-SwiftUI, Deploy-UIKit, Deploy-Common]
```